### PR TITLE
fix(eosio): update to newer version

### PR DESCRIPTION
**Pull request by leisterfrancisco**
Update to newer eosio version and reorder project file structure

### CODE REVIEW

File: .github/workflows/publish-docker-image.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+      
+      - name: Extract metadata (tags, labels) for Docker
+        id: meta
+        uses: docker/metadata-action@v3
+        with:
+          images: |
+            eoscostarica506/eosio-local
```
Comment (xavier506) [inline .github/workflows/publish-docker-image.yaml:36]: should image name be `eos-local` not `eosio-local`?

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+# build contracts
+RUN git clone https://github.com/EOSIO/eosio.contracts.git
+RUN cd /app/eosio.contracts \
+  && git checkout release/1.8.x \
+  && ./build.sh -c /usr/local/eosio.cdt
+
+FROM eosio/eosio:release_1.8.x as nodeos
+WORKDIR /app
+ENV TESTNET_EOSIO_PRIVATE_KEY 5KQwrPbwdL6PhXujxW37FSSQZ1JiwsST4cqQzDeyXtP79zkvFD3
```
Comment (xavier506) [inline Dockerfile:21]: should these keys be  different than from other local environments? maybe they can be configures as env vars

File: README.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+</p>
+
+# EOS Mainnet Local Network
+
+## Description
+EOS Local provides a quick way to setup an EOS local network for development.
+
+The primary benefits of containers are consistency across different environments and deployment ease.
+
+This project has a [full-stack-boilerplate](https://github.com/eoscostarica/full-stack-boilerplate) for your dApp.
```
Comment (xavier506) [inline README.md:18]: This project works along with a [full-stack-boilerplate](https://github.com/eoscostarica/full-stack-boilerplate) to help you build your EOS dApp.

File: README.md [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+
+Finally, a factor to consider is the time that is reduced in the initial configuration of any network, this image allows directly, with only two commands to have the network installed and ready to perform functionality tests as necessary.
+
+### Contracts
+The EOSIO image is based on the `eosio.system`, `eosio.token` and `eosio.msig` contracts for its configuration. Your code can be found at [this link](https://github.com/EOSIO/eosio.contracts/tree/master/contracts).
+1. **eosio.system**: Defines the structures and actions needed for blockchain's core functionality.
+2. **eosio.token**: Defines the structures and actions that allow users to create, issue, and manage tokens for EOSIO-based blockchains.
+3. **eosio.msig**: Allows the creation of proposed transactions that require authorization from a list of accounts.
+
```
Comment (xavier506) [inline README.md:32]: should we load the eden contracts also?
simpleassets / atomicassets/ o dgoods ?

Comment (leisterfrancisco) [reply README.md:32]: I don't thinks so but we could create a section next to it talking about some projects that can be deployed on it

File: .github/workflows/publish-docker-image.yaml (hunk 1 above)
Comment (leisterfrancisco) [reply .github/workflows/publish-docker-image.yaml:36]: sorry, my bad, eos-local is righ

Comment (leisterfrancisco) [reply .github/workflows/publish-docker-image.yaml]: Fixed here: https://github.com/eoscostarica/eos-local/pull/88/files#diff-ac9bda107c3925408bf0ffb31e643a4d1e8b9953abc242d8fdd4b32b0a68263aR36

File: README.md (hunk 4 above)
Comment (leisterfrancisco) [reply README.md:32]: Fixed here: https://github.com/eoscostarica/eos-local/pull/88/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R33

**Review by xavier506 — APPROVED** [review]
LGTM

**Review by xavier506 — APPROVED** [review]
LGTM

**Pull request merged**